Bump SEV Crate to fix ARM build issue

fitzthum opened the pull request:
Test https://github.com/virtee/sev/pull/329

fitzthum commented:
Compilation error seems to be good https://github.com/confidential-containers/trustee/actions/runs/17279027197/job/49043047711?pr=931

Xynnn007 commented:
Good. https://github.com/virtee/sev/pull/329 has been merged. Let's move it forward

fitzthum commented:
ok, we know have a `6.3.1` release so I've updated the Cargo file and rebuilt the lock and things work. (I dropped the test commit from this PR, but I also ran it through the CI with that commit). Technically this PR might not be necessary since Cargo will probably pick up the point release on its own. Still, it's good to rebuild the lockfile. Fortunately the az crates are happy with the point release as well. 

cc @Xynnn007 @mythi

mythi reviewed (approved):
Thanks!
